suite-native/app/index.js

```diff
@@ -1,3 +1,4 @@
+import 'react-native-get-random-values';
```

Comment:
Could not we use AI for this? 😅

Just joking. But why is this needed? You are not importing anything specific from this package. Is it some pollyfil that Solana uses in the background?

Reply:
It has react native implementation of `crypto.getRandomValues()`. Otherwise Solana api wont work

Reply:
Maybe it should not be placed in the metro.config.ts, then. We are already polyfilling the `crypto` module there. Cc @Nodonisko, what is your opinion about it?

Reply:
If we need only that one function we already using expo-crypto in app which has getRandomValues too https://docs.expo.dev/versions/latest/sdk/crypto/#cryptogetrandomvaluesttypedarray

Reply:
I just need get polyfill for crypto.getRandomValues and this one does just this one thing, has bunch of installs (900k) and is suggested by solana mobile for expo: https://docs.solanamobile.com/react-native/expo

Reply:
updated in c92279d039

Reply:
Fyi: I have a followup branch to get SOL data for `getAccountHistoryMovementFromTransactions`, but I dont want to make this PR too complex